packages/passage/src/stimulus-tabs.jsx

```diff
@@ -15,11 +14,10 @@ const styles = (theme) => ({
   },
   tab: {
     backgroundColor: color.background(),
-    fontSize: '0.8125em',
+    fontSize: theme.typography.fontSize,
```

Comment:
please test what happens if font-size on html is 10px, and on body is14px. I have the impression that it will not change accordingly to this both. Test both for tabs an text